src/main/java/org/sonarlint/intellij/config/global/ServerConnectionMgmtPanel.java

```diff
@@ -144,11 +226,14 @@ public JComponent getComponent() {
 
   @Override
   public boolean isModified(SonarLintGlobalSettings settings) {
-    return !connections.equals(settings.getServerConnections());
+    getComponent();
+    return !connections.equals(settings.getServerConnections()) || settings.isFocusOnNewCode() != focusOnNewCode.isSelected();
   }
 
   @Override
   public void save(SonarLintGlobalSettings newSettings) {
+    getComponent();
```

Comment:
I've seen that we also call `getComponent()` in other classes but is it required?

Reply:
It creates the component if it does not exist, I didn't think too much about it but I think it can be removed